autogluon/inference/docker/0.4/py3/Dockerfile.cpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,182 @@
+{
+  "com.google.code.gson:gson": [
+    {
+      "reason_to_ignore": "Ray 1.x does not have a version with patched gson; upgrading to 2.x would require a migration + snakeyaml is not patched in any version; ray security team was emailed: https://github.com/ray-project/ray/pull/29165",
+      "description": " The package com.google.code.gson:gson before 2.8.9 are vulnerable to Deserialization of Untrusted Data via the writeReplace() method in internal classes, which may lead to DoS attacks.",
+      "vulnerability_id": "CVE-2022-25647",
+      "name": "CVE-2022-25647",
+      "package_name": "com.google.code.gson:gson",
+      "package_details": {
+        "file_path": "usr/local/lib/python3.8/dist-packages/ray/jars/ray_dist.jar",
+        "name": "com.google.code.gson:gson",
+        "package_manager": "JAR",
+        "version": "2.8.6",
+        "release": null
+      },
+      "remediation": {
+        "recommendation": {
+          "text": "None Provided"
+        }
+      },
+      "cvss_v3_score": 7.5,
+      "cvss_v30_score": 0.0,
+      "cvss_v31_score": 7.5,
+      "cvss_v2_score": 5.0,
+      "cvss_v3_severity": "HIGH",
+      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-25647.html",
+      "source": "UBUNTU_CVE",
+      "severity": "MEDIUM",
+      "status": "ACTIVE",
+      "title": "CVE-2022-25647 - com.google.code.gson:gson"
+    },
+    {
+      "reason_to_ignore": "Ray 1.x does not have a version with patched gson; upgrading to 2.x would require a migration + snakeyaml is not patched in any version; ray security team was emailed: https://github.com/ray-project/ray/pull/29165",
+      "description": "Affected versions of this package are vulnerable to Deserialization of Untrusted Data via the `writeReplace()` method in internal classes, which may lead to DoS attacks.",
+      "vulnerability_id": "IN1-JAVA-COMGOOGLECODEGSON-1730327",
+      "name": "IN1-JAVA-COMGOOGLECODEGSON-1730327",
+      "package_name": "com.google.code.gson:gson",
+      "package_details": {
+        "file_path": "usr/local/lib/python3.8/dist-packages/ray/jars/ray_dist.jar",
+        "name": "com.google.code.gson:gson",
+        "package_manager": "JAR",
+        "version": "2.8.6",
+        "release": null
+      },
+      "remediation": {
+        "recommendation": {
+          "text": "None Provided"
+        }
+      },
+      "cvss_v3_score": 7.7,
+      "cvss_v30_score": 0.0,
+      "cvss_v31_score": 7.7,
+      "cvss_v2_score": 0.0,
+      "cvss_v3_severity": "HIGH",
+      "source_url": "https://snyk.io/vuln/SNYK-JAVA-COMGOOGLECODEGSON-1730327",
+      "source": "SNYK",
+      "severity": "HIGH",
+      "status": "ACTIVE",
+      "title": "SNYK-JAVA-COMGOOGLECODEGSON-1730327 - com.google.code.gson:gson"
```

Comment:
Can we please add a key called `reason_to_ignore`: "xyz..."?

Reply:
As seen here: https://github.com/aws/deep-learning-containers/pull/2328/files#diff-67be49f98eac4ee50ac13b54139abc30c2451fe163e30d876c3d09849c0a9a16R30

Reply:
I just realized that we have added `reason_to_ignore` at the top. Will it be possible to place it at the bottom to maintain consistency while reading/reviewing?

Reply:
Updated `reason_to_ignore` to be at the bottom.